bilingual-module/README

```diff
@@ -26,49 +27,60 @@ if_hfst}}
 
 If this does not make any sense, we recommend you look at: www.apertium.org.
 
+
 Compiling
 -------------------------------------------------------------------------------
 
 Given the requirements being installed, you should be able to just run:
 
-    $ ./configure
-    $ make
-    # make install
+```bash
+$ ./configure
+$ make
+$ make install
```

Comment:
The `#` is intentional. It means run the command as root.

Reply:
On a related note, does this actually render differently with the language hint?

Reply:
Yeah, for example this is without the language hint:

![before](https://user-images.githubusercontent.com/22768990/47603029-ff17e500-d9ee-11e8-94c2-afd7af148925.png)

And this is with the hint:

![after](https://user-images.githubusercontent.com/22768990/47603032-06d78980-d9ef-11e8-92c4-e3aa427ed3be.png)

(Screenshots are from the Markdown Preview in Atom, which should be using (more or less) the same renderer as the Markdown renderer GitHub uses.)